### DISCUSSION TITLE
Photoprism won't start

### DISCUSSION THREAD

**Discussion by alanpaone** [photoprism/photoprism#2034] [Q&A] [answered]
Both the preview and latest tags won't start in docker (on a synology NAS if it helps). MariaDB works fine, but photoprism stops almost immediately after starting. I've tried moving pointing my docker compose at empty folders to start fresh, but it did not make a difference. here's the full log from photoprism. 

```
started as root

init tensorflow-amd64-avx2...

curl -fsSL "https://dl.photoprism.app/tensorflow/linux/libtensorflow-linux-avx2-1.15.2.tar.gz" > /tmp/libtensorflow-linux-avx2-1.15.2.tar.gz

tar --overwrite -C "/usr" -xzf /tmp/libtensorflow-linux-avx2-1.15.2.tar.gz

ldconfig

running as root

photoprism start

/entrypoint.sh: line 94:    19 Illegal instruction     (core dumped) "$@"
```

**Comment by Genygo** [comment, marked as answer]
Did you have looked at the docs? There is a chapter regarding Synology NAS [here](https://docs.photoprism.app/getting-started/nas/synology/)
When your device is ARMv7 you have to use the 32bit Docker images

**Reply by lastzero** [reply]
The problem here is that tensorflow-amd64-avx2 was added/initialized via configuration option, but the CPU does not have AVX2 support. Reset the Docker image and use the TensorFlow libraries included by default.

**Reply by Genygo** [reply]
missed that, thanks for pointing this out!

**Reply by alanpaone** [reply]
its not arm, its an Intel processor and i probably should have mentioned its been working for about a year and only stopped recently

**Reply by Genygo** [reply]
have you set `PHOTOPRISM_INIT` with any value in your `docker-compose.yml`?

**Reply by alanpaone** [reply]
I forgot to reply to this, but changing photoprism_init to "intel-graphics" ended up working. I'm not sure why AVX2 used to work, but this was the right track, so thanks!
